cypress/support/commands.js

```diff
@@ -1,25 +1,28 @@
-// ***********************************************
-// This example commands.js shows you how to
-// create various custom commands and overwrite
-// existing commands.
-//
-// For more comprehensive examples of custom
-// commands please read more here:
-// https://on.cypress.io/custom-commands
-// ***********************************************
-//
-//
-// -- This is a parent command --
-// Cypress.Commands.add('login', (email, password) => { ... })
-//
-//
-// -- This is a child command --
-// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
-//
-//
-// -- This is a dual command --
-// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
-//
-//
-// -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+import jobData from '../fixtures/job-data.json';
+import modifiedJob from '../fixtures/modified-job.json';
+import singleJob from '../fixtures/single-job.json';
+
+const baseURL = 'https://spacetruckersapi.herokuapp.com/api/v1/jobs'
+const jobURL = 'https://spacetruckersapi.herokuapp.com/api/v1/jobs/1'
+const jobDeleteURL = 'https://spacetruckersapi.herokuapp.com/api/v1/jobs/2'
+
+Cypress.Commands.add('stubAllJobs', () => {
+  cy.intercept(baseURL, jobData)
+    .visit('http://localhost:3000/')
+});
+
+Cypress.Commands.add('patchJob', () => {
+  cy.intercept('PATCH', jobURL , modifiedJob)
+});
+
+Cypress.Commands.add('removeJob', () => {
+  cy.intercept('DELETE', jobDeleteURL , singleJob)
+});
+
+Cypress.Commands.add('modifiedJob', () => {
+  cy.intercept(baseURL, modifiedJob)
+});
+
+// Cypress.Commands.add('getSingleJob', () => {
+//   cy.intercept(baseURL , singleJob)
+// });
```

Comment:
We will probably want to remove this commented out code

Reply:
Added change